invokeai/backend/model_manager/config.py

```diff
@@ -186,7 +194,9 @@ def json_schema_extra(schema: dict[str, Any], model_class: Type[BaseModel]) -> N
 class CheckpointConfigBase(ModelConfigBase):
     """Model config for checkpoint-style models."""
 
-    format: Literal[ModelFormat.Checkpoint] = ModelFormat.Checkpoint
+    format: Literal[ModelFormat.Checkpoint, ModelFormat.BnbQuantizednf4b] = Field(
+        description="Format of the provided checkpoint model", default=ModelFormat.Checkpoint
+    )
```

Comment:
Is `format` the right field to use? Currently `format` answers "is this a single-file model or is this a diffusers-style model". Is `ModelFormat.BnbQuantizednf4b` really a model format?

Reply:
A few lines below, it looks like a model can have both a `type` and `format` of `T5Encoder`, but the `format` could also be `T5Encoder8b`. It's confusing to me.

Reply:
Yeah this is what I meant when I mentioned format not feeling like the right field for saying whether a model was quantized or not. They really should be installed with different loaders though and the loader is decided by model type, base type, and format. Wasn't sure trying to change that was a good use of time right now

Reply:
Got it, shouldn't be too hard to migrate if we want to change this in the future